app/views/admin/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="admin-menu-container">
+  <div class="menu-side">
+      <div class="nav">
+        <!-- In the span will be the number of citisienz and takecares peoeple -->
+        <a href="admin/index">Adulto Mayor</a>
+        <p>14</p>
+      </div>
+      <div class="nav">
+        <a href="admin/caretaker_index">Cuidador</a>
+        <p>2</p>
+      </div>
+  </div>
+  <div class="form-side">
+  <%= render 'partials/form_profile' %>
```

Comment:
Can you indent these partials please. If they belong inside the `form-side` `div` then they should be indented.